feat: improve the website with AI 🤖 (#495)
countr/website: content/docs/01-features/10-timeouts.md

# Timeouts

A timeout role is a way for you to automatically give a role to users who fail a certain number of times within a certain time period. This can be useful for preventing spam in your channel or slowing down the rate of failed counts.

## Purpose

> The main purpose of a timeout role is to prevent people from spamming your channel. By automatically giving a role to users who fail a certain number of times within a certain time period, you can restrict that role from sending messages in the counting channel. This can help slow down the rate of failed counts and keep your channel more organized.

Reviewer: We know this, this whole section feels redundant but I know it's the style for the feature pages :p
```suggestion
By automatically giving a role to users who fail a certain number of times within a certain time period, you can restrict that role from sending messages in the counting channel. This can help slow down the rate of failed counts and keep your channel more organized.
```

## How to set up {#how}

> To configure a timeout role, use the `/timeoutrole set` command. This command allows you to set the number of failed counts and the time period in which those counts must occur. Once you've configured a role, you can do whatever you want with it. The normal use case is to deny the "Send Messages" permission in the counting channel, so that users with the timeout role cannot send messages in that channel.

Reviewer: ```suggestion
To configure a timeout role, use the `/timeoutrole set` command. This command allows you to set the number of failed counts and the time period in which those counts must occur. Once you've configured a role, you can do whatever you want with it. Typically, you will deny the "Send Messages" permission in the counting channel, so that users with the timeout role cannot send messages in that channel.
```

You can also configure a flow to do something whenever someone gets a timeout role. For example, you could send a message to a log channel or notify a moderator.

To remove the timeout role, use the `/timeoutrole remove` command.

## How to check if you're timed out {#check}

To check if you're timed out, use the `/user` command. This command will tell you if you're timed out and how much time is left in your timeout period. If you're not timed out, the command will not mention your timeout status.

:::note
The /user command only mentions your timeout status if you're actually timed out. If you're not timed out, the command will not mention your timeout status.

> :::

Reviewer: Also weird AI rambling, I can't leave a suggestion because it selects a removed section but consider removing the note and just adding:

"If you're not timed out, the command will not mention your timeout status." to the paragraph above.
